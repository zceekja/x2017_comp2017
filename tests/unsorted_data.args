Wirragulla Cardiff "10:00:00"