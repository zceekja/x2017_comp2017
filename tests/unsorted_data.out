The next train to Cardiff from Wirragulla departs at 10:01:23
